box-share-sdk/build.gradle

```diff
@@ -50,12 +49,12 @@ repositories {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:26.1.0'
-    implementation 'com.android.support:design:26.1.0'
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'com.google.android.material:material:1.0.0'
     // If using local dependency instead of maven, use the path pointing to the content sdk to replace
     // the maven dependency. e.g.:compile project(':box-android-content-sdk:box-content-sdk')
 
-    api 'com.box:box-android-sdk:4.99.697-SNAPSHOT'
+    api 'com.box:box-android-sdk:5.99.702-SNAPSHOT'
```

Comment:
Could we replace this with release version ?

Reply:
Sure, I will replace that after new release of Android Content SDK

Reply:
I would suggest first doing release of Android Content SDK, then following with others which depend on it.